.gitignore

```diff
@@ -0,0 +1,8 @@
+checklist.md
+*.sqlite3
+.idea/
+README.md
+venv/
+tests/
+**__pycache__/
+*.pyc
```

Comment:
Add empty line

It is a nit, but remember to do this every time you're creating a new file. Then Github won't highlight this line with red circle